### PR TITLE
Document DropBox on untested Windows.

### PR DESCRIPTION
**Pull request by ximenesuk**
This PR supports the changes in https://github.com/openmicroscopy/openmicroscopy/pull/3361

I'd welcome comments on wording, location, and mark-up!

### CODE REVIEW

File: omero/sysadmins/dropbox.txt [hunk 1]
```diff
@@ -38,6 +39,12 @@ the following more specific requirements:
     operating system. Watching a network-attached share (NAS) is strictly
     ***not*** supported.
 
+.. note:: It is likely, but not guaranteed, that DropBox will run on later
+    versions of Windows, however only tested versions are supported by default.
+    Setting the Windows system environment variable  ``OMERO_PLATFORM_CHECK``
```
Comment (mtbc) [inline omero/sysadmins/dropbox.txt]: double space; also, `:envvar:`?

File: omero/sysadmins/dropbox.txt [hunk 2]
```diff
@@ -38,6 +39,12 @@ the following more specific requirements:
...
+    to ``false`` and restarting the server will allow DropBox to be started on
+    an untested version of Windows.
```
Comment (mtbc) [inline omero/sysadmins/dropbox.txt]: some kind of, "Let us know if it works" ?

**Comment by manics** [comment]
Looks fine in a local build... unless you want to take this opportunity to clean up the whole page?

**Comment by ximenesuk** [comment]
If you feel the whole page needs cleaning up please don't let me stop you :-)

File: omero/sysadmins/dropbox.txt [hunk 3]
```diff
@@ -38,6 +39,15 @@ the following more specific requirements:
...
     ***not*** supported.
 
+.. note:: It is likely, but not guaranteed, that DropBox will run on later
+    versions of Windows, however only tested versions are supported by default.
+    Setting the Windows system environment variable
+    :envvar:`OMERO_PLATFORM_CHECK` to ``false`` and restarting the server will
+    allow DropBox to be started on an untested version of Windows.
+
+    If you do successfully run DropBox on an as yet untested version of Windows
+    please do let us know.
```
Comment (hflynn) [inline omero/sysadmins/dropbox.txt]: Would probably be nice to add 'via the :community_plone:`mailing lists or forum  <>`. ' just for completeness.

Comment (ximenesuk) [inline omero/sysadmins/dropbox.txt]: Added. Thanks.

**Comment by hflynn** [comment]
:+1:

**Comment by hflynn** [comment]
Now built on staging, looks good to me, merging unless you have any further comments @manics ?

**Comment by manics** [comment]
Nothing to add

**Comment by hflynn** [comment]
Rebase to develop please, thanks.

**Comment by ximenesuk** [comment]
--rebased-to #1082

**Pull request merged**